CMakeLists.txt

```diff
@@ -16,3 +16,5 @@ include_directories(${hiredis_INCLUDE_DIR})
 
 add_subdirectory (src)
 add_subdirectory (test)
+
+add_definitions(-cpp)
```

Comment:
is this the best way to add the C preprocessor to a cmake build?

Reply:
I think if we change the extension from `.f90` to `.F90` the fortran compiler will know automatically to run the preprocessor.